core/src/test/java/nl/nn/adapterframework/jdbc/dbms/TestBlobs.java

```diff
@@ -147,25 +170,25 @@ public void testWriteAndReadBlobUsingDbmsSupport(int numOfBlocks, int blockSize)
 
 	}
 
-	public void testWriteAndReadClobUsingDbmsSupport(int numOfBlocks, int blockSize) throws Exception {
+	public void testWriteAndReadClobUsingDbmsSupport(int numOfBlocks, int blockSize, DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
 		String block = getBigString(1,blockSize);
-		String insertQuery = "INSERT INTO "+TEST_TABLE+" (TKEY,TCLOB) VALUES (20,?)";
-		String selectQuery = "SELECT TCLOB FROM "+TEST_TABLE+" WHERE TKEY=20";
-		try (PreparedStatement stmt = connection.prepareStatement(insertQuery)) {
-			Object clobInsertHandle = dbmsSupport.getClobHandle(stmt, 1);
-			try (Writer clobWriter = dbmsSupport.getClobWriter(stmt, 1, clobInsertHandle)) {
+		String insertQuery = "INSERT INTO " + TABLE_NAME + " (TKEY,TCLOB) VALUES (20,?)";
+		String selectQuery = "SELECT TCLOB FROM " + TABLE_NAME + " WHERE TKEY=20";
+		try (PreparedStatement stmt = databaseTestEnvironment.getConnection().prepareStatement(insertQuery)) {
+			Object clobInsertHandle = databaseTestEnvironment.getDbmsSupport().getClobHandle(stmt, 1);
```

Comment:
Omdat je `dbmsSupport` heel vaak nodig hebt zou je die even in een lokale variabele kunnen zetten misschien.